client/webserver/webserver_test.go

```diff
@@ -390,7 +390,7 @@ func TestClientMap(t *testing.T) {
 
 	// Close the server and make sure the connection is closed.
 	shutdown()
-	time.Sleep(time.Millisecond)
+	time.Sleep(10 * time.Millisecond)
```

Comment:
Could we do a StartStopWaiter or a manual WaitGroup around go s.sRun in newTServer?

It's certainly more convenient to just sleep a bit, but ideally we'd sync even in tests.

Reply:
Oh, I just looked more closely at `(*WebServer).Run`... thoughts about having Run wait internally for it's closer goroutine before returning?

Reply:
BTW, I can't repro it locally either.  CI pushes our assumptions, and that's probably helpful if annoying.